README.md

```diff
@@ -2,6 +2,16 @@
 
 Web application that connects programmers based on their experience, interests and the type of person they are looking for (mentor, buddy, mentee) through matching and contact sharing.
 
+## Prerequisites ⚙️
+
+-   [Node.js](https://nodejs.org/en/download/) with `npm` package manager
```

Comment:
When we're using `Yarn` as a package manage, do we need to mention `npm`? I know it comes with `Node.js`, but this sounds like we're using two package managers at once. :thinking:

Reply:
I understand this argument. It is true that this information is useless here. And maybe confusing. Good catch :+1: :wink: